src/test/java/org/simulator/fba/BiGGTest.java

```diff
@@ -61,36 +66,16 @@ public BiGGTest(String resource) {
 
 		// find all BiGG models (compressed .xml.gz files)
         String biggPath = getBiGGModelPath();
-        System.out.println("BiGG models path: " + biggPath);
-		return TestUtils.findResources(biggPath, ".xml.gz", filter, skip, mvnResource);
+        logger.info("BiGG models path: " + biggPath);
+		return TestUtils.findResources(biggPath, ".xml", filter, skip, mvnResource);
```

Comment:
This looks awkward. Please adjust this TestUtils method to throw a corresponding exception, e.g. if the resource is missing.

Reply:
Here, I think there is no exception. Whenever there are no files with this extension, it sends empty Iterable object.